maintain.js

```diff
@@ -163,9 +163,12 @@ module.exports = {
         if (process.cwd() + '/' + 'package.json' == filePath) {
           dataForChecks.packageName = fileContent.name
           try {
-            dataForChecks.orgName = fileContent.repository.url.split('/')[3]
+            let repo_url_rx =
+              /(git@|(git|(git\+)*https):\/\/)github.com(\/|:)[a-z]+\/[a-z|-]+(.git)*/
+            dataForChecks.orgName =
+              fileContent.repository.url.match(repo_url_rx)[5]
           } catch (error) {
-            dataForChecks.orgName = '+++illegalchar+++'
+            dataForChecks.orgName = null
```

Comment:
don't use a try-catch for this - keep it for actual exceptions
regex.match returns `undefined` if no match, so just check for that

Reply:
Grand, it's been replaced with match()?.[5] set the value to undefined if needs be without having a second check